### DISCUSSION TITLE
does the Realtime work on voice calls

### DISCUSSION THREAD

**Discussion by Cxsmo-ai** [voicepaw/so-vits-svc-fork#148] [Q&A] [answered]
does the Realtime work on voice calls

**Comment by ItsMe-TJ** [comment, marked as answer]
It should yeah! I tried it out with virtual audio cable and I see no reason why It wouldn't work.

**Reply by ruckusmattster** [reply]
how do you setup virtual audio cable?

**Reply by ADR3-N** [reply]
If anyone still has this question, this is where I got mine. It was very simple, install, restart, and you should see the driver as an option immediately after
https://vb-audio.com/Cable/

**Comment by ainub** [comment]
For windows the previous answer works, I thought I would provide a solution for linux:
1. Create a VirtualSink and a Virtual Microphone with Pulseaudio helpers (works on pipewire system):
```
# Create a virtual sink
pactl load-module module-null-sink sink_name=VirtualSpeaker sink_properties=device.description=VirtualSpeaker
# Link it with a virtual microphone that is visible in apps
pactl load-module module-null-sink media.class=Audio/Source/Virtual sink_name=VirtualMic channel_map=front-left,front-right
pw-link VirtualSpeaker:monitor_FL VirtualMic:input_FL
pw-link VirtualSpeaker:monitor_FR VirtualMic:input_FR
```

2. Start svc vc with your params
3. Using pavucontrol you can now change the output of svc to the VirtualSpeaker and select VirtualMic in Discord or your application
